05-onchain.md

```diff
@@ -185,231 +161,192 @@ Similarly, when node A sees a *commitment transaction* from B:
 
-1. The other node is not irrevocably committed to it; this can only
-   happen if we have not received `revoke_and_ack` so that the other
-   node holds two valid commitment transactions, one with the HTLC and
-   one without.  We won't normally know the preimage here, unless it's
-   a payment to ourselves, and revealing that would be an information leak,
+1. The offerer is not irrevocably committed to the offered HTLC; this can only
+   happen if the recipient has not received `revoke_and_ack` so that the offerer
+   holds two valid commitment transactions, one with the HTLC and
+   one without. The recipient won't normally know the preimage here, unless it's
+   a payment to themselves, and revealing that would be an information leak,
    so it's best to allow the HTLC to time out in this case.
-2. The other node is irrevocably committed to it, but we haven't yet
-   committed to an outgoing HTLC.  In this case we can either forward
+2. The offerer is irrevocably committed to the offered HTLC, but the recipient hasn't yet
+   committed to an outgoing HTLC. In this case the recipient can either forward
    or timeout.
-3. We have committed to an outgoing HTLC for this incoming one.  In
-   this case we have to use the preimage if we receive it from the
-   outgoing HTLC, otherwise we will lose funds by making an outgoing
+3. The recipient has committed to an outgoing HTLC for the offered one. In
+   this case the recipient has to use the preimage if they receive it from the
+   outgoing HTLC, otherwise they will lose funds by making an outgoing
    payment without redeeming the incoming one.
 
 
 ## Requirements
 
-
 If the node receives (or already knows) a payment preimage for an
 unresolved HTLC output it was offered for which it has committed to an
 outgoing HTLC, it MUST *resolve* the output by spending it.
 Otherwise, if the other node is not irrevocably committed to the HTLC,
 it MUST NOT *resolve* the output by spending it.
 
-
 To spend an offered HTLC output: if the transaction is the node's own commitment transaction, then it MUST use the HTLC-success transaction, and the
 HTLC-success transaction output MUST be *resolved* as described in
 "On-chain HTLC Transaction Handling", otherwise, it MUST spend the output to a convenient address.
 
-
 If not otherwise resolved, once the HTLC output has expired, it is considered
 *irrevocably resolved*.
 
-
 ## Rationale
 
-
-If this is our commitment transaction, we can only use a payment
+If this is the recipient's commitment transaction, they can only use a payment
 preimage with the HTLC-success transaction (which preserves the
-`to_self_delay` requirement).  Otherwise we can create any transaction we want to
+`to_self_delay` requirement). Otherwise the recipient can create any transaction they want to
 resolve it.
 
-
-We don't care about expired offers: we should have ensured that the
+The recipient doesn't care about expired offers: they should have ensured that the
```

Comment:
...offers: it's the sender's responsibility to ensure that...

Reply:
Changed.